src/app/tests/suites/certification/Test_TC_SC_5_2.yaml

```diff
@@ -90,7 +90,11 @@ tests:
       command: "writeAttribute"
       attribute: "GroupKeyMap"
       arguments:
-          value: [{ FabricIndex: 1, GroupId: 0x0103, GroupKeySetID: 0x01a3 }]
+          value:
+              [
+                  { FabricIndex: 0, GroupId: 0x0101, GroupKeySetID: 0x01a3 },
+                  { FabricIndex: 0, GroupId: 0x0103, GroupKeySetID: 0x01a3 },
```

Comment:
Followup: can we fix our codegen so leaving off FabricIndex here is ok?

Reply:
It is not needed in the python version. So I guess we can left it as if until we switch.

Reply:
Sounds good.